js/app.js

```diff
@@ -1072,6 +1074,9 @@ const main = () => {
           $scope.lyricArray = parseLyric(lyric);
         });
         $scope.lastTrackId = data;
+        const { ipcRenderer } = require('electron');
+        console.log(JSON.stringify(track.title));
```

Comment:
please remove console.log after debug

Reply:
> please remove console.log after debug

Thanks for reminding. Already removed now.